docusaurus/docs/operate/run_a_node/full_node_walkthrough.md

```diff
@@ -295,6 +296,39 @@ Or, using curl:
 curl -X GET http://localhost:26657/block | jq
 ```
 
+#### How do I access my CometBFT endpoint externally?
+
+The default CometBFT port is at `26657`.
+
+To make it accessible externally, you'll need to port all the instructions from
+port `26656` on this page to port `26657`. Specifically:
+
+```bash
+# Update your firewall
+sudo ufw allow 26657/tcp
+
+# Update your iptables
+sudo iptables -A INPUT -p tcp --dport 26657 -j ACCEPT
+
+# Update your Cosmovisor config
+sed -i 's|laddr = "tcp://127.0.0.1:26657"|laddr = "tcp://0.0.0.0:26657"|' $HOME/.poktroll/config/config.toml
+sed -i 's|cors_allowed_origins = \[\]|cors_allowed_origins = ["*"]|' $HOME/.poktroll/config/config.toml
+
+# Restart the service
+sudo systemctl restart cosmovisor.service
+
+# Test the connection
+nc -vz $VULTR_GROVE_TEST_IP 26657
```

Comment:
```suggestion
nc -vz {EXTERNAL_IP} 26657
```

In other similar places in this document, we use `{EXTERNAL_IP}`. I'd keep it for consistency.